include/net/lwm2m.h

```diff
@@ -318,6 +319,76 @@ struct coap_block_context *lwm2m_firmware_get_block_context();
 #endif
 #endif
 
+
+#if defined(CONFIG_LWM2M_SWMGMT_OBJ_SUPPORT)
```

Comment:
I think these should be in a separate header, no?

Reply:
No this is how all the existing standard objects (such as FOTA ) does it